app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt

```diff
@@ -238,7 +239,7 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
             R.id.menu_force_landscape -> toggleOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
             R.id.menu_default_orientation -> toggleOrientation(ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED)
             R.id.menu_save_as -> saveImageAs()
-            R.id.menu_create_shortcut -> createShortcut()
+            R.id.menu_create_shortcut -> if (isOreoPlus()) createShortcut()
```

Comment:
keep the checks in the code please for simplicity, the logic is never here at onOptionsItemSelected

Reply:
Done https://github.com/SimpleMobileTools/Simple-Gallery/pull/2291/commits/0e1a1cfb90acd875cdbc9a9046156a677ba96cd6